src/raftstore/store/peer.rs

```diff
@@ -1348,7 +1349,8 @@ impl Peer {
             "peer_id" => self.peer.get_id(),
             "update" => %update,
         );
-        if let Err(e) = local_reader.schedule(update) {
+        let l = hash_u64(self.region_id) as usize % local_readers.len();
```

Comment:
How about moving the `mod` op into `hash_u64`? It is dangerous to schedule msgs to a wrong localreader.